src/Php85/Uri/Rfc3986Uri.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Symfony\Polyfill\Php85\Uri;
+
+use Symfony\Polyfill\Php85\Exception\ParsingException;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ *
+ * @internal
+ */
+class Rfc3986Uri extends Uri
```

Comment:
This needs to extend `Uri\Uri` instead of extending the stub abstract class directly. Otherwise, the stub `Uri\Rfc3986Uri` class will not extend `Uri\Uri`